content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -148,37 +186,38 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ```
 
-You cannot modify which `Role` or `ClusterRole` a binding object refers to.
-Attempts to change the `roleRef` field of a binding object will result in a validation error.
-To change the `roleRef` field on an existing binding object, the binding object must be deleted and recreated.
-There are two primary reasons for this restriction:
+After you create a binding, you cannot change the Role or ClusterRole that it refers to.
+If you try to change a binding's `roleRef`, you get a validation error. To get the outcome
+you wanted, you would need to remove the binding object and create a replacement.
+
+There are two core reasons for this restriction:
 
+1. Making `roleRef` immutable allows granting someone `update` permission on an existing binding
+object, so that they can manage the list of subjects, without being able to change
+the role that is granted to those subjects.
 1. A binding to a different role is a fundamentally different binding.
 Requiring a binding to be deleted/recreated in order to change the `roleRef`
 ensures the full list of subjects in the binding is intended to be granted
-the new role (as opposed to enabling accidentally modifying just the roleRef 
+the new role (as opposed to enabling accidentally modifying just the roleRef
 without verifying all of the existing subjects should be given the new role's permissions).
-2. Making `roleRef` immutable allows giving `update` permission on an existing binding object
-to a user, which lets them manage the list of subjects, without being able to change the 
-role that is granted to those subjects.
 
 The `kubectl auth reconcile` command-line utility creates or updates a manifest file containing RBAC objects,
 and handles deleting and recreating binding objects if required to change the role they refer to.
 See [command usage and examples](#kubectl-auth-reconcile) for more information.
 
-### Referring to Resources
+### Referring to resources
 
-Most resources are represented by a string representation of their name, such as "pods", just as it
-appears in the URL for the relevant API endpoint. However, some Kubernetes APIs involve a
-"subresource", such as the logs for a pod. The URL for the pods logs endpoint is:
+In the Kubernetes API, most resources are represented and accessed using a string representation of
+their object name, such as `pods` for a Pod. Some Kubernetes APIs involve a
+_subresource_, such as the logs for a pod. A request for the pods logs endpoint looks like:
```

Comment:
This concept could perhaps do with a term. If I could say “the _foo_ name for that resource” then it might make the documentation easier to clarify: “set this to `resources: ["somethings"]` because the _foo_ name for SomeThing resources is `somethings`”.

(but if there isn't already a commonly used term, we'll have to live without it).